pyeapi/api/ospf.py

```diff
@@ -90,6 +94,19 @@ def _parse_ospf_process_id(self, config):
         match = re.search(r'^router ospf (\d+)', config)
         return dict(ospf_process_id=int(match.group(1)))
 
+    def _parse_vrf(self, config):
+        """Parses config file for the OSPF vrf name
+
+           Args:
+               config(str):  Running configuration
+           Returns:
+               dict: key: ospf_vrf (str)
+        """
+        match = re.search(r'^router ospf \d+ vrf (.*)', config)
```

Comment:
`.*` is generally a scary combo :-) Would it be possible to \w or something that would best match the vrf string name?